src/game.js

```diff
@@ -0,0 +1,44 @@
+"use strict"
+
+
+function Game(){
+
+	this.bowlingFrames = [];
+	this.score = 0;
+	this.scoresArray = [];	
+
+};
+
+
+Game.prototype.playFrame = function() {
+	let frame = new Frame;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing '()' invoking a constructor.<br>'Frame' is not defined.